ui/page/settings_page.go

```diff
@@ -479,7 +510,10 @@ func (pg *SettingsPage) Handle() {
 			pg.showSPVPeerDialog()
 			return
 		}
-		pg.WL.MultiWallet.DeleteUserConfigValueForKey(specificPeerKey)
+
+		title := "Remove specific peer"
+		msg := "Are you sure you want to proceed with removing the specific peer?"
+		pg.showWarningModalDialog(title, msg, specificPeerKey)
```

Comment:
Is there any need for adding a prompt here? It doesn't look necessary to confirm from the user before removing persistent peer.

Reply:
I am looking at the point where a user accidentally clicks on the button. The peer is removed almost immediately and the user would need to set it up the second time.